imagetest/test_suites/storageperf/iops_write_test.go

```diff
@@ -16,8 +16,10 @@ import (
 )
 
 const (
```

Comment:
A lot of these parameters are repeated in each definition, is there any way to separate this into standardFioOptions, randomFioOptions, sequentialFioOptions, etc and just append each list of options when we need it in getLinuxWriteOptions?

Reply:
Ideally, do the for read options if possible

Reply:
I refactored some helper methods so that it uses common options and methods for most of the test